README.md

```diff
@@ -29,3 +29,33 @@ What is the asymptotic runtime ($\Theta$) of your algorithm in terms of the
 number of comparisons? What is it in terms of the number of flips? Add your
 answer to this markdown file.
 
+## My Runtime Analysis
+
+When pancakeSort() is called, it enters a for loop that will iterate through
+ the array backwards. This operation will be a complexity of $\Theta(n)$. 
+
+Inside this loop is the function findMaxIndex() which looks through a sub-array 
+of the array given. The time complexity of this function is is $\Theta(n)$.
+The flip() method is also called within the loop, and there are two for loops that 
+are called. The first one going through the sub-array and reversing the elements. 
+The second one then replaces these element in the original array. Both of these 
+loops combines will take a time complexity of $\Theta(2n)$ (using $m$ for the same 
+reasoning as with the findMaxIndex() method). There are then two flip() calls 
+inside the main loop, which increases the complexity to $\Theta(4n)$.
+
+Combining these complexities, we get $\Theta(n*5n)$. Ignoring constants, this 
+can simplify down to $\Theta(n*n)$ or $\Theta(n^2)$ for the final complexity.
```

Comment:
So what is the complexity in terms of number of comparisons and number of flips?

Reply:
(I see that I accidently left in some lines (line 42 & 43), those parentheses are not meant to be there.)

I'm not sure I understand... Could you elaborate a little more? From what I understand, the flip() function uses two for loops, each a complexity of Θ(n). Added together, resulting in Θ(2n). There two flip() calls which takes a complexity of Θ(4n).

Reply:
If you don't count operations, but the number of flips and comparisons. E.g. there might be $O(n^2)$ flips.